frontend/public/src/lib/notificationsApi.js

```diff
@@ -56,6 +57,11 @@ export function parseStoredUserMessage(
     msgText =
         "We're sorry, your country is currently blocked from enrolling in this course."
     break
+  case USER_MSG_TYPE_COURSE_NON_UPGRADABLE:
+    alertType = ALERT_TYPE_DANGER
+    msgText =
+        "The upgrade deadline for a course in your cart has passed. Please try later."
```

Comment:
This message could be interpreted as meaning the learner could try tomorrow or next week, which won't be the case.

Maybe having it be something like:

> The upgrade deadline for a course in your cart has passed. You can retake and pay for this course the next time it runs.

CC @pdpinch

Reply:
This probably makes more sense, I've updated this text.